dist/phenogrid-bundle.css

```diff
@@ -422,8 +422,6 @@ td,
 th {
   padding: 0;
 }
-
```

Comment:
I didn't merge these CSS changes, I'll make more changes in CSS files very soon to add some prefix to ids and classes to avoid possible naming conflicts. E.g. "pg_"